#!/usr/bin/env python
PACKAGE = "car_detect"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("min_cluster_size",	int_t,	0,	"Lower bound for number of points in a cluster", 100, 0, 10000)
gen.add("alpha", 	double_t,	0,	"Upper bound for abs(cosOXY) for X Y to be points of the same object", 0.9, 0.5, 0.999)
gen.add("alpha_ground", double_t, 0, "Upper bound for abs(cosOXY for Y to be point of the road if X is such one", 0.8, 0.5, 0.999)
gen.add("detections_filter_upper_bound", double_t, 0, "Max size for bbox", 4, 1, 10)
exit(gen.generate(PACKAGE, "tracker_fast", "general"))
